lib/auth/2fa.ts

```diff
@@ -1,13 +1,13 @@
 import { NotifyClient } from "notifications-node-client";
 import { logMessage } from "@lib/logger";
-import crypto from "crypto";
+import { generateTokenCode } from "@lib/auth/tokenGenerator";
 const TEMPLATE_ID = process.env.TEMPLATE_ID;
 const NOTIFY_API_KEY = process.env.NOTIFY_API_KEY;
 
-export const generateVerificationCode = () => {
+export const generateVerificationCode = async () => {
   // Temporary - for testing purposes
```

Comment:
I am guessing this could now be removed. I can do it as part of my next 2FA pull request.